tests/integration/suite/test_node.py

```diff
@@ -160,13 +160,22 @@ def _node_template():
     node_template = wait_for(_node_template,
                              fail_handler=lambda:
                              'failed to create node template')
-    wait_remove_resource(node_template)
```

Comment:
This should stay put, because if a lower line fails for some reason it still gets cleaned up post-method completion. If this is below that might not happen. It won't remove it early.